logger.ts

```diff
@@ -4,6 +4,6 @@ import config from './server/config'
 
 const formatOut = bunyanFormat({ outputMode: 'short', color: !config.production })
 
-const logger = bunyan.createLogger({ name: 'Hmpps Ciag Careers Induction Ui', stream: formatOut, level: 'debug' })
+const logger = bunyan.createLogger({ name: 'HMPPS CIAG Careers Induction UI', stream: formatOut, level: 'debug' })
```

Comment:
This is the standard logger from the bootstrap template; I've tidied/corrected the casing of the service name that is logged.

All changes below that change imports are changing from using the duplicate logger to this one